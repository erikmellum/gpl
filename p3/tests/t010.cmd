-s 42 tests/t010.gpl
