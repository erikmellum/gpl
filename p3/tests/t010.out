gpl.cpp::main()
  input file(tests/t010.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(false)
  print_symbol_table(false)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

1 error found.
gpl giving up.
